cfpb/cfgov-refresh
# Add Wagtail module for Social Media molecule

This PR enables the Social Media molecule to be used as a module in Wagtail, available to the Sidefoot.


## Additions

- Ability to add Social Media module to Sidefoot streams in Wagtail


## Changes

- Tweak styling of the Social Media molecule


## Testing

1.  Pull down this branch
1. `gulp styles`
1. `./cfgov/manage.py migrate`
1. `./cfgov/manage.py runserver`
1. Login to your local Wagtail at http://localhost:8000/admin/
1. Add the new Social Media module to a page's Sidefoot
1. Preview that page
1. Also review an existing Blog page and the Footer to ensure the Social Media molecule there looks as expected.


## Review

- @anselmbradford 
- @KimberlyMunoz 
- @sebworks 
- @kave 


## Screenshots

![screen shot 2016-05-20 at 10 06 10](https://cloud.githubusercontent.com/assets/1044670/15430264/8dd0ebec-1e72-11e6-98ff-29254a2ea093.png)


## Notes

- I added 30px of top margin to `.post_body`, because bottom margin was removed from the Social Media molecule's atoms to be consistent to the atomic design spacing principles as I (and @anselmbradford) understand them. 30px seems consistent with the latest designs for articles, though those haven't been updated in quite a while. Please comment if you have any concerns with this. /cc @duelj @ajbush 


## Checklist
